src/pybel/io/mongodb.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+"""An exporter from PyBEL graphs to a local Mongo database."""
+
+import logging
+from copy import deepcopy
+from typing import Any, List, Mapping, Optional, Tuple
+
+from pymongo import MongoClient
+from pymongo.collection import Collection
+
+from pybel import BELGraph, to_sbel
+from pybel.dsl import Entity, Variant
+from pybel.schema import is_valid_edge, is_valid_node
+
+__all__ = [
+    '_entity_to_dict',
+    '_rm_mongo_keys',
+    'to_mongodb',
+    'find_nodes',
+    'get_edges_from_node',
+    'get_edges_from_criteria'
+]
+
+logger = logging.getLogger(__name__)
+
+
+def _entity_to_dict(entity: Entity) -> Mapping[str, Any]:
```

Comment:
why is this necessary? isn't the Entity already a dict?

Reply:
Yep, that makes sense. I just need it in testing to make sure the node is equal to the Mongo entry, so the two need to be of the exact same type. I'll relocate it to the test file for clarity.